Removed customFonts feature flag

minimaluminium opened the pull request:
ref DES-949

minimaluminium committed:
Removed customFonts feature flag

minimaluminium committed:
Updated a snapshot

aileen commented on ghost/core/test/unit/frontend/helpers/body_class.test.js:
```diff
@@ -217,8 +215,6 @@ describe('{{body_class}} helper', function () {
         });
 
         it('does not include custom font classes when custom fonts are not enabled', function () {
```
This test is fully redundant now, so I'll remove it.

aileen committed:
Fixed failing unit tests for `{{body_class}}` helper

- With the removal of the flag, some of the tests were missing the object path and throw an error
- Added optional chaining to ensure the logic doesn't throw errors when a `site` object is not available

aileen committed:
Updated `{{ghost_head}}` snapshots to include `data-locale` after i18N changes
